test/controllers/users/users.createAdmin.spec.js

```diff
@@ -0,0 +1,158 @@
+const request = require('supertest');
+const app = require('../../../app');
+const UserModel = require('../../../app/models').user;
+const { updateUser } = require('../../../app/services/userService');
+
+describe('POST /admin/users', () => {
+  let jwtToken = null;
+
+  beforeEach(async done => {
+    const { body } = await request(app)
+      .post('/users/signup')
+      .send({
+        firstName: 'John',
+        lastName: 'Doe',
+        email: 'johndoe@wolox.com.ar',
+        password: 'Sherman33'
+      });
+    await updateUser(body.user.id, { role: 'admin' });
+
+    await request(app)
+      .post('/users/signup')
+      .send({
+        firstName: 'Normal',
+        lastName: 'User',
+        email: 'normal@wolox.com.ar',
+        password: 'Sherman33'
+      });
+
+    const loginAdminRequest = await request(app)
+      .post('/users/sessions')
+      .send({
+        email: 'johndoe@wolox.com.ar',
+        password: 'Sherman33'
+      });
+    jwtToken = loginAdminRequest.body.token;
+    done();
+  });
```

Comment:
This is quite heavy for a test, and you are coupling the test with signup and session controllers.
To avoid this you could use a factory for the db resources (create the users) and a mock for the session.